src/audiomixerboard.cpp

```diff
@@ -42,6 +42,7 @@ CChannelFader::CChannelFader ( QWidget* pNW ) :
     pPan                        = new QDial       ( pLevelsBox );
     pPanLabel                   = new QLabel      ( tr ( "Pan" ), pLevelsBox );
     pInfoLabel                  = new QLabel      ( "", pLevelsBox );
+    pInfoLabel->setMinimumSize(36, 36); // prevents jitter when muting/unmuting
```

Comment:
Please follow the coding standards - spaces before and after braces.  (I think also a blank line before line 44 to keep these two lines grouped would be good, too.)